tests/test_ote_training.py

```diff
@@ -0,0 +1,613 @@
+        self.openvino_task = openvino_task_cls(self.environment_for_export)
+
+        logger.debug('Get predictions on the validation set')
+        validation_dataset = self.dataset.get_subset(subset)
+        self.predicted_validation_dataset_exp = self.openvino_task.infer(
+            validation_dataset.with_empty_annotations(),
+            InferenceParameters(is_evaluation=True))
+        self.resultset_exp = ResultSetEntity(
+            model=self.exported_model,
+            ground_truth_dataset=validation_dataset,
+            prediction_dataset=self.predicted_validation_dataset_exp,
+        )
+        logger.debug('Estimate quality on validation set')
+        self.openvino_task.evaluate(self.resultset_exp)
+        self.evaluation_performance_exported = self.resultset_exp.performance
+
+        logger.info(f'performance exported={self.evaluation_performance_exported}')
+        score_name, score_value = performance_to_score_name_value(self.evaluation_performance_exported)
+        data_collector.log_final_metric('evaluation_performance_exported/' + score_name, score_value)
+        return self.evaluation_performance_exported
+
+# Note that each TestBunch describes a group of similar tests
+# If 'model_name' or 'dataset_name' are lists, cartesian product of tests will be run.
+TestBunch = namedtuple('TestBunch', ['model_name', 'dataset_name', 'num_training_iters', 'usecase'])
+
+# pytest magic
+def pytest_generate_tests(metafunc):
+    if metafunc.cls is None:
+        return
+    if not issubclass(metafunc.cls, TestOTETraining):
+        return
+
+    # It allows to filter by usecase
+    usecase = metafunc.config.getoption('--test-usecase')
+
+    argnames, argvalues, ids = metafunc.cls.get_list_of_tests(usecase)
+    metafunc.parametrize(argnames, argvalues, ids=ids, scope='class')
+
+class TestOTETraining:
+    PERFORMANCE_RESULTS = None # it is required for e2e system
+
+    DEFAULT_NUM_ITERS = 1
+    test_bunches = [
+#           TestBunch(
+#               model_name=[
+#                   'face-detection-0200',
+#                   'face-detection-0202',
+#                   'face-detection-0204',
+#                   'face-detection-0205',
+#                   'face-detection-0206',
+#                   'face-detection-0207',
+#               ],
+#               dataset_name='airport_faces',
+#               num_training_iters=None,
+#               usecase='precommit',
+#           ),
+#           TestBunch(
+#               model_name=[
+#                   'horizontal-text-detection-0001',
+#               ],
+#               dataset_name='horizontal_text_detection',
+#               num_training_iters=None,
+#               usecase='precommit',
+#           ),
+            TestBunch(
+                model_name=[
+                   'mobilenetV2_ATSS',
+                   'mobilenetV2_SSD',
+                   'resnet50_VFNet'
+                ],
+                dataset_name='vitens_tiled_shortened_500_A',
```

Comment:
Is it Ok to be here?

Reply:
It is an inner dataset, but it is used in internal tests only. 
I could fix it and use a public dataset in one of the next PR-s.

Reply:
Fixed